css/app.css

```diff
@@ -23,19 +23,38 @@ gaia-list h3 {
   width: calc(100% - 10px);
 }
 
-gaia-list .action {
-  text-align: center;
-  text-decoration: underline;
-  background-color: var(--button-background);
-}
-
 gaia-list .action.disabled {
   pointer-events: none;
   color: var(--header-disabled-button-color);
 }
 
-gaia-list .destructive {
-  color: var(--color-destructive);
+.flex {
+  display: flex;
+  flex-flow: row;
+  justify-content: center;
+  align-items: center;
+  padding: 0.5rem;
+}
+
+gaia-button.top {
+  margin: 0 !important;
+  padding: 0.5rem;
+}
+
+gaia-button.bottom {
+  min-width: 90% !important;
+  margin: 0 !important;
+  padding: 0.5rem;
+}
+
```

Comment:
Are the `!important`s needed because of a component issue?
If so let's add a comment explaining it (maybe with a bug number referenced :)).